### DISCUSSION TITLE
Google OAuth enabled

### DISCUSSION THREAD

**Discussion by peekeesh** [damongolding/immich-kiosk#64] [Q&A] [answered]
Will this work with Google Oauth? I have Google Oauth enabled and can't seem to get this to work. I might be missing a step or two or is it unsupported at the moment?

Thanks!

**Comment by damongolding** [comment]
I don't believe having Oauth enabled on Immich should affect the API endpoints (but I may be wrong)

1. Which version of Kiosk are you running?

2. Do you see any errors in the log output?

**Reply by peekeesh** [reply]
Hello,

I just deployed the instance using docker compose. So i would assume i have the latest version.

When i check the log there is no error. When i go to the url, i get a message that says unauthorized.But i maybe doing something wrong here... 

Thanks!

![image](https://github.com/user-attachments/assets/f30fc916-76df-49bb-8072-5fedd68566ce)

**Reply by damongolding** [reply, marked as answer]
Do you have the `password` ENV or yaml field set to anything?
